sdk/identity/identity/test/internal/node/msalClient.spec.ts

```diff
@@ -140,33 +140,6 @@ describe("MsalClient", function () {
         const config = msalClient.generateMsalConfiguration(clientId, tenantId, {});
         assert.instanceOf(config.system!.networkClient, IdentityClient);
       });
-
-      it("configures logging options", function () {
```

Comment:
This was superfluous, we already have a test that exercises logging end-to-end and tests that a logger can be passed